notebooks/105-language-quantize-bert/105-language-quantize-bert.ipynb

```diff
@@ -7,8 +7,8 @@
     "id": "rQc-wXjqrEuR"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.    print(f'F1 score: {metric}')
</pre><p>possibly it can make sense to add formating with 2 digits after coma</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/941/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>